content/docs/user-guide/testing/getting-started/_index.md

```diff
@@ -0,0 +1,232 @@
+
+To verify everything is set up correctly, run the [CMake configure command](/docs/user-guide/build/configure-and-build/) from the CMake CLI or CMake GUI. This will register everything you just added, and emit errors if anything was misconfigured.
+
+#### Step 3: Write new tests
+
+Now that you have configured CMake to create a library from your tests, and registered this library with CTest, you are ready to write new tests. To simplify your module structure, create new test files inside `o3de/.../<MyModule>/tests/`
+
+Tests are written using standard [GoogleTest](https://github.com/google/googletest/blob/main/docs/index.md) syntax, which helps you write small functions to test your code. To pull in everything from GoogleTest plus a few convienient tools, add the following statement to your file:
+
+```cpp
+#include <AzTest/AzTest.h>
+```
+
+To keep test functions legible at a glance, we recommend using the [Osherove Naming Convention](https://osherove.com/blog/2005/4/3/naming-standards-for-unit-tests.html) of `UnitOfWork_StateUnderTest_ExpectedBehavior`. This can be especially helpful when reading a report with many individual test case failures. In a short a C++ example:
+
+```cpp
+TEST_F(Matrix4x4Tests, MatrixMultiply_InverseMatrix_ReturnIdentityMatrix)
+{
+    // (function under test would be called here)
+    ASSERT_TRUE(someErrorState);
+    EXPECT_TRUE(someResult);
+    EXPECT_FALSE(secondaryProperty);
+}
+```
+
+For more information on writing tests in C++, see [Using AzTest](/docs/user-guide/testing/aztest/aztest/).
+
+## GoogleBenchmark
+
+For performance benchmarks of small pieces of C++ code O3DE uses [GoogleBenchmark](https://github.com/google/benchmark/blob/main/docs/index.md), which is quite similar to GoogleTest. However how failure is defined is a major difference between a normal test and a benchmark. In most tests, pass/fail status is directly evaluated into a boolean state. Benchmarks instead create a subjective performance metric. These metrics are most valuable when periodically recorded, to help detect trends. The only objective failure during a benchmark occurs when the code fails to run or crashes.
+
+The steps for configuring a GoogleBenchmark library are identical to the [steps above for GoogleTest libraries](#googletest), with a couple exceptions. The include statement is `#include <benchmark/benchmark.h>` and the cmake helper function is:
+
+```
+ly_add_googlebenchmark(
+    NAME MyModule.Benchmarks
+)
+```
+
+## PyTest
+
+Certain tests are easier to write in a scripting language like Python. The scope of these tests are typically at the [integration level](https://softwaretestingfundamentals.com/integration-testing/), verifying mid-to-high level functionality of workflows and overall system sanity. Additionally, any Python-based tools will prefer unit tests written in Python. For these tests O3DE uses [PyTest](https://docs.pytest.org/).
```

Comment:
```suggestion
Certain tests are easier to write in a scripting language like Python. The scope of these tests are typically at the [integration level](https://softwaretestingfundamentals.com/integration-testing/), verifying mid-to-high level functionality of workflows and overall system soundness. Additionally, any Python-based tools will prefer unit tests written in Python. For these tests O3DE uses [PyTest](https://docs.pytest.org/).
```

Reply:
"Sanity" is a common term in the testing community, whereas "soundness" is surprising to me. Is there a reason to prefer this change? Does the term "sanity" need to be better defined?

Reply:
I think that would help; as a reader, I'm unsure how aspects of an overall system sanity check differ from a check of mid-to-high level functionality of workflows.